content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,17 +8,22 @@ menu:
 weight: 101
 ---
 
-## v2.0.0-beta.11 [2020-05-??]
+## v2.0.0-beta.11 [2020-05-22]
 
 ### Features
-- Integrate UTC dropdown when making custom time range query
-- Allow for min OR max y-axis visualization settings rather than min AND max
-- Add CSV to line protocol conversion library
+
+- Ability to set UTC time for a custom time range query.
+- Ability to set a minimum or maximum value for the y-axis visualization setting (rather than requiring both).
+- New `csv2lp` library for converting CSV (comma separated values) to InfluxDB line protocol.
 
 ### Bug Fixes
-- Fixed bug that wasn't persisting `timeFormat` for Graph + Single Stat selections.
-- Authorizer now exposes full permission set
-- Fixed issue that was causing variable selections to hydrate all variable values.
+
+- Automatically adjust the drop-down list width to ensure the longest item in a list is visible.
+- Fix bug in Graph + Single Stat visualizations to ensure `timeFormat` persists.
+- Authorizer now exposes the full permission set. This adds the ability to derive which organizations the Authorizer has access to read or write to without using a User Request Management (URM) service.
+- Fix issue causing variable selections to hydrate all variable values, decreasing the impact on network requests.
```

Comment:
@Asalem1 is "decreasing the impact on network requests." accurate to say here?

Reply:
Ya. Having said that I just undid some of the work since it uncovered a couple of bugs in the way variables are implemented on the UI. It’s still safe to say, but way more limited in its capacity than it had been originally